rust/src/serialization.rs

```diff
@@ -3208,7 +3407,7 @@ impl DeserializeEmbeddedGroup for OperationalCert {
 
 impl cbor_event::se::Serialize for HeaderBody {
     fn serialize<'se, W: Write>(&self, serializer: &'se mut Serializer<W>) -> cbor_event::Result<&'se mut Serializer<W>> {
-        serializer.write_array(cbor_event::Len::Len(15))?;
+        serializer.write_array(cbor_event::Len::Len(14))?;
```

Comment:
The change to `impl cbor_event::se::Serialize for HeaderBody {` here breaks backwards compatibility for this library